src/back/TicketManager/tests/PaymentService/TicketManager.PaymentService.ServicesTests/Endpoints/Payments/ListPaymentsEndpointTests.cs

```diff
@@ -0,0 +1,29 @@
+using FastEndpoints;
+using FluentAssertions;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using Moq.EntityFrameworkCore;
+using TicketManager.PaymentService.Domain.Payments;
+using TicketManager.PaymentService.Services.DataAccess;
+using TicketManager.PaymentService.Services.Endpoints.Payments;
+using Xunit;
+
+namespace TicketManager.PaymentService.ServicesTests.Endpoints.Payments;
+
+public class ListPaymentsEndpointTests
+{
+    [Fact]
+    public async Task WhenItIsCalled_ItShouldAddPayment()
+    {
+        
+        List<Payment> payments = new List<Payment> {new(), new(), new()};
+        
+        var dbContextMock = new Mock<PaymentDbContext>(new DbContextOptionsBuilder<PaymentDbContext>().Options);
+        dbContextMock.Setup(d => d.Payments).ReturnsDbSet(payments);
+        var dbContext = dbContextMock.Object;
+        var endpoint = Factory.Create<ListPaymentsEndpoint>(dbContext);
+        await endpoint.HandleAsync(default);
+        var result = endpoint.Response;
+        result.Payments.Count.Should().Be(payments.Count);
```

Comment:
```suggestion
        result.Payments.Should().BeEquivalentTo(payments);
```

Reply:
Or actually something similar, cause `Payment` and `PaymentDto` have different fields. But I guess my suggestion should work just like that.